.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
@tfyh Your files in the .idea directory shouldn't be pushed to github. Please add a ".gitignore" file in the main directory and exlcude ".idea/" in it. Please commit the .gitignore file (push it with your PR) and make sure that none of your .idea files are part of this PR.